extra/2-piping.js

```diff
@@ -16,26 +16,27 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+  return a + b;
 }
 
-function multiply() {
-
+function multiply(a, b) {
+  return a * b;
 }
 
-function format() {
 
+function format(num) {
+  return `£${num}`
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+let badCode = `£${(startingValue + 10 ) * 2}`
 
```

Comment:
My explanation was:

 //Multiple methods are used in one line which makes it hard to determine the results from each methods execution. It is confusing and hard to follow.